### DISCUSSION TITLE
Subversion bottle doesn't work on M1 and Monterrey

### DISCUSSION THREAD

**Discussion by swimboy** [Homebrew/discussions#2463] [Everyday usage]
I have a MacBook Pro with an M1Pro processor and tried to install svn. After it installed, when I run svn, I get the error message: 
dyld[27042]: Symbol not found: _apr_crypto_block_cleanup
  Referenced from: /opt/homebrew/Cellar/subversion/1.14.1_4/lib/libsvn_subr-1.0.dylib
  Expected in: /usr/lib/libaprutil-1.0.dylib

Any suggestions on how to fix this?

**Comment by carlocab** [comment]
What is the output of `brew config` and `brew doctor`, `brew linkage subversion`, and `brew info apr-util`?

**Reply by EliTayrien** [reply]
I had the same error. I didn't see anything odd in `brew config` or `brew doctor`, but `brew linkage subversion` showed:

```
Broken dependencies:
  /usr/local/opt/apr-util/lib/libaprutil-1.0.dylib (apr-util)
```


In my case, I was able to resolve the issue by running `arch -x86_64 brew reinstall apr-util` and `arch -x86_64 brew reinstall subversion` . After running these commands, the dependency was no longer listed as missing in `linkage`, and `svn --version` prints `svn, version 1.14.1 (r1886195)` as expected.

**Reply by leafac** [reply]
I’m running into the same error. Funny thing is that I don’t even want to use SVN itself, but it’s a requirement to install some fonts 🙃

I tried your workaround, but it doesn’t work because I’m running Homebrew ARM:

```
$ arch -x86_64 brew reinstall apr-util && arch -x86_64 brew reinstall subversion && brew install font-ibm-plex-sans
Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.
```

Installing Homebrew x64 seems too big of a ask to install a font, so I’m not doing it right now…
